### ISSUE TITLE
Growling libnotify on Ubuntu

### ISSUE THREAD

**Issue by jsuereth** [softprops/sbt-growl-plugin#4]
Hey guys,

I got the sbt-growl-plugin working on Ubuntu using libnotify-bin (notify-send).   A real solution would use a JVM dbus library, but this works great for now.   

The project is here: https://github.com/jsuereth/sbt-growl-plugin

I updated the README for Ubuntu.  What I'm curious about is whether or not I _broke_ Mac growling.   Also, my personal preference for growling is to place in `~/.sbt/plugins/build.sbt` and _NOT_ specify _anything_ in project-specific config (although it'd be neat if including an icon in project-specific config location would enable it's usage in growling...).   This is my huge annoyance with the posterous plugin right now on your project.  Because I'm running SBT nightlies for the Scala SBT build, I had to un-configure posterous (it can't find un-published versions).   If you like this work, you can always enable posterous by removing the commit where I undo posterous.

The other thing I did was tweak how icons are configured/chosen _and_ I added a default Scala logo that is used if no icons are configured.  It's a bit of a hack, but it works really nicely on my system.

Let me know what you think of the changes.  I'm using the raw source reference in my `~/.sbt/plugins/projcet/Build.scala` and it's working beautifully on ubuntu.  Would love to see if it still works for Mac and your thoughts on my commits.   

So to recap, my changes:
- Added Ubuntu support
- Added default Scala logo to use if no other icon found.
- Moved default icon path to `~/.sbt/growl/icons`
- Defaulted icons to look in the default path for `pass.png`, `fail.png` and `error.png`
- Modified settings so they are automatically applied to project.   Ideal for `~/.sbt/plugins/build.sbt` configuration.

If you like what I've done, I'll issue a pull request, otherwise let me know what to fix up first :)
